scene/resources/default_theme/default_theme_builders.py

```diff
@@ -0,0 +1,41 @@
+"""Functions used to generate source files during build time
+
+All such functions are invoked in a subprocess on Windows to prevent build flakiness.
+
+"""
+import os
+import os.path
+from platform_methods import subprocess_main
+
+
+def make_fonts_header(target, source, env):
+
+    dst = target[0]
+
+    g = open(dst, "w", encoding="utf-8")
+
+    g.write("/* THIS FILE IS GENERATED DO NOT EDIT */\n")
+    g.write("#ifndef _EDITOR_FONTS_H\n")
+    g.write("#define _EDITOR_FONTS_H\n")
```

Comment:
```suggestion
    g.write("#ifndef _DEFAULT_FONTS_H\n")
    g.write("#define _DEFAULT_FONTS_H\n")
```

Probably define name should be different, to avoid potential conflict with actual editor fonts (and match the filename).